app/src/main/res/layout/settings.xml

```diff
@@ -68,71 +73,79 @@
                 android:requiresFadingEdge="horizontal"
                 android:scrollbars="none">
 
-                <LinearLayout
+                <RelativeLayout
                     android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:orientation="horizontal">
+                    android:layout_height="wrap_content">
 
-                    <RelativeLayout
-                        android:layout_width="@dimen/settings_width"
-                        android:layout_height="wrap_content">
+                    <LinearLayout
+                        android:id="@+id/first_row"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:orientation="horizontal">
 
-                        <Space
-                            android:id="@+id/center"
-                            android:layout_width="0dp"
-                            android:layout_height="0dp"
-                            android:layout_centerHorizontal="true"
-                            android:visibility="invisible" />
+                        <org.mozilla.vrbrowser.ui.views.HoneycombButton
+                            android:id="@+id/displayButton"
+                            style="?attr/settingsButtonStyle"
+                            android:layout_marginRight="10dp"
```

Comment:
can you adjust the vertical centreing of this one?

![image](https://user-images.githubusercontent.com/203725/48245807-9a229e80-e3a1-11e8-9a0b-5f0b78a40eba.png)

it'd be nice if all the icons are lined up on the bottom row of honeycombs too. thanks in advance!

Reply:
Fixed buttons alignments.